### PR TITLE
fix page unresponsiveness while selecting '1 Month' in the Analytics

### PR DESCRIPTION
**Pull request by Yibaebi**

### CODE REVIEW

File: app/components/partner/client-uploads-stat-chart/index.hbs [hunk 1]
```diff
@@ -48,13 +48,16 @@
               class='button'
               type='button'
               local-class='option {{if
-                (eq this.currentTimeline.key option.key)
```
Comment (cosmosgenius) [inline app/components/partner/client-uploads-stat-chart/index.hbs:51]: This should still be key, as the point of key is to do object matching.

File: app/components/partner/client-uploads-stat-chart/index.js [hunk 2]
```diff
@@ -65,10 +69,12 @@ export default class PartnerClientUploadsStatChartComponent extends Component {
       },
     },
     {
-      key: this.intl.t('monthWithNum', {
+      key: 'month',
+      axisText: this.intl.t('month'),
+      buttonSelectorText: this.intl.t('monthWithNum', {
```
Comment (cosmosgenius) [inline app/components/partner/client-uploads-stat-chart/index.js:74]: can you change this to the new month inlt.

**Review by cosmosgenius — CHANGES_REQUESTED** [review]

File: app/components/partner/client-uploads-stat-chart/index.js (hunk 2 above)
Comment (Yibaebi) [reply app/components/partner/client-uploads-stat-chart/index.js:74]: Are you referring to the axisText?

File: app/components/partner/client-uploads-stat-chart/index.hbs (hunk 1 above)
Comment (Yibaebi) [reply app/components/partner/client-uploads-stat-chart/index.hbs:51]: Done

File: app/components/partner/client-uploads-stat-chart/index.js (hunk 2 above)
Comment (Yibaebi) [reply app/components/partner/client-uploads-stat-chart/index.js]: Done

**Review by cosmosgenius — APPROVED** [review]

**Pull request merged**